app/javascript/application.js

```diff
@@ -1,3 +1,13 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+import React from "react";
+import ReactDOM from "react-dom";
+import Greetings from "./components/Greetings";
+function App() {
+  return (
+    <div>
+      <Greetings />
```

Comment:
- [ ] It will be great to ensure that your greeting component is set up as a route. In this implementation you are doing is just rendering your greeting component. Remember there should be a route to this component. ⬇️ 
![Screenshot 2023-05-04 at 12 27 35](https://user-images.githubusercontent.com/32361963/236165928-bc1fb609-7689-4906-9b2c-ec4d6da19bf3.png)